lib/assert.js

```diff
@@ -342,7 +342,7 @@ function _throws(shouldThrow, block, expected, message) {
   actual = _tryBlock(block);
 
   message = (expected && expected.name ? ' (' + expected.name + ').' : '.') +
-            (message ? ' ' + message : '.');
+            (message ? ' ' + message : '');
```

Comment:
Yes, unless `message` is expected to already end with `.`, so ¯\\_(ツ)_/¯. And there should also be a `:` rather than `.` after `expected` if `message` is truthy.

Thinking about this some more and looking at the uses, I wonder if the thing to do is get rid of all the logic for appending `.` in the first place. Nothing else appends `.`. And this mostly only seems to do it wrong!

Current master results in messages like this:

```console
AssertionError: Missing expected exception..
AssertionError: Missing expected exception. foo bar baz
```

wat?

Compare to `assert.strictEqual()` which does not append `.`:

```console
AssertionError: 'a' === 'b'
AssertionError: foo bar baz
```